build/mvn

```diff
@@ -68,7 +68,7 @@ install_app() {
 # Install maven under the build/ folder
 install_mvn() {
   install_app \
-    "http://apache.claz.org/maven/maven-3/3.2.3/binaries" \
+    "http://archive.apache.org/dist/maven/maven-3/3.2.3/binaries" \
```

Comment:
Yeah, if there are any redirectors, I'd like to use it but I also don't know so I chose `archive.apache.org`. For now, it's reliable.